### PR TITLE
refactor(rust): kafka-* commands

### PR DESCRIPTION
**Pull request by adrianbenavides**
- Add missing CRUD commands
- Add JSON output + improve the plain output
- Add bats tests for the basic operations

### CODE REVIEW

File: implementations/rust/ockam/ockam_api/src/nodes/models/services.rs [hunk 1]
```diff
@@ -237,6 +237,14 @@ impl Display for ServiceStatus {
 
 impl Output for ServiceStatus {
     fn item(&self) -> crate::Result<String> {
-        Ok(format!("{}", self))
+        let mut f = String::new();
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_api/src/nodes/models/services.rs:240]: What is the difference with `Ok(format!("{}{}", fmt::PADDING, self))`?

File: implementations/rust/ockam/ockam_command/src/kafka/inlet/create.rs [hunk 2]
```diff
@@ -31,46 +31,57 @@ use crate::{
     Command, CommandGlobalOpts,
 };
 
-/// Create a new Kafka Inlet. Kafka clients v3.7.0 and earlier are supported. You can find the version you have with 'kafka-topics.sh --version'.
+/// Create a new Kafka Inlet.
+/// Kafka clients v3.7.0 and earlier are supported.
+/// You can find the version you have running 'kafka-topics.sh --version'.
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/src/kafka/inlet/create.rs]: `You can find the version you have with 'kafka-topics.sh --version'.` to stay consistent with `consumer/create`

File: implementations/rust/ockam/ockam_command/src/kafka/outlet/create.rs [hunk 3]
```diff
@@ -49,10 +55,16 @@ impl Command for CreateCommand {
...
-        opts.terminal
-            .write_line(&fmt_log!("Creating KafkaOutlet service"))?;
-        let is_finished = Mutex::new(false);
-        let send_req = async {
+
+        let outlet = {
+            let pb = opts.terminal.progress_bar();
+            if let Some(pb) = pb.as_ref() {
+                pb.set_message(format!(
+                    "Creating Kafka Outlet to bootstrap broker {}...\n",
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/src/kafka/outlet/create.rs]: "bootstrap server", to stay consistent, maybe?

File: implementations/rust/ockam/ockam_command/tests/bats/local/kafka.bats [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  assert_output 2
+  # Check the address of the inlets
+  run_success $OCKAM kafka-inlet list --jq '.[].addr'
+  assert_output --partial "kafka_inlet"
+  assert_output --partial "inlet2"
+
+  # Delete the first inlet
+  run_success $OCKAM kafka-inlet delete kafka_inlet --yes
+
+  # Check that there are no inlets
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/tests/bats/local/kafka.bats]: Shouldn't that comment say "there is only one inlet left"?

**Review by etorreborre — DISMISSED** [review]

File: implementations/rust/ockam/ockam_api/src/nodes/models/services.rs (hunk 1 above)
Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_api/src/nodes/models/services.rs:240]: Pretty much the same, given we add the `\n` to the `format` function. I used the `writeln` pattern just to keep it consistent with the rest of Output/Display implementations.

File: implementations/rust/ockam/ockam_command/tests/bats/local/kafka.bats (hunk 4 above)
Comment (adrianbenavides) [reply implementations/rust/ockam/ockam_command/tests/bats/local/kafka.bats]: Copy-paste error, fixed!

**Review by etorreborre — APPROVED** [review]

**Pull request merged**